paperweight-core/src/main/kotlin/taskcontainers/GeneralTasks.kt

```diff
@@ -50,4 +50,13 @@ open class GeneralTasks(
         inputJar.set(extractFromBundler.flatMap { it.serverJar })
         includes.set(extension.vanillaJarIncludes)
     }
+
+    val collectAtsFromPatches by tasks.registering<CollectATsFromPatches> {
```

Comment:
Is there a reason these are in general tasks and not just in alltasks? beyond that this looks good

Reply:
Well, the merged paper ats file is needed in SpigotTasks. The tasks didn’t feel right there since they have nothing to do with spigot.

Reply:
ah nvm I see now